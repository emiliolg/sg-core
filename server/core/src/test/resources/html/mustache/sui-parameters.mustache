<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views" sg-params="text:String, number:Int, decimal:Decimal, bool:Boolean, list:String*">
<head>
    <title>Parameters examples</title>
</head>
<body>

<div>Text: {{text}}</div>
<div>Number: {{number}}</div>
<div>Decimal: {{decimal}}</div>
<div>Currency: {{#$}}{{decimal}}{{/$}}</div>
<div>Bool: {{#bool}}{{bool}}{{/bool}}</div>


<div>Bool Condition: {{#bool}}{{bool}}{{/bool}}</div>
<div>Bool Inverse Condition: {{^bool}}{{bool}}{{/bool}}</div>
<div>String list: {{#list}}{{this}}{{^-last}}, {{/-last}}{{/list}}</div>

</body>
</html>
