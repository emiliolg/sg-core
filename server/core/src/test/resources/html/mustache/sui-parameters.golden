<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views" sg-params="text:String, number:Int, decimal:Decimal, bool:Boolean, list:String*">
<head>
    <title>Parameters examples</title>
</head>
<body>

<div>Text: ñandú</div>
<div>Number: 2</div>
<div>Decimal: 1.123</div>
<div>Currency: $1.12</div>
<div>Bool: true</div>


<div>Bool Condition: true</div>
<div>Bool Inverse Condition: </div>
<div>String list: one, two, three</div>

</body>
</html>
